property.go

```diff
@@ -8,7 +8,11 @@ import (
 // Property value belongs to an error instance only, never inherited from a type.
 // Property visibility is hindered by Wrap, preserved by Decorate.
 type Property struct {
-	id    int64
+	*property
+}
+
+type property struct {
+	id    uint64
 	label string
```

Comment:
It looks like we never use the label at this moment. If we remove it, the property key may be a simple unique id wrapper.

Reply:
I want to to add "info property" in next PR - property that will be pushed to Error message.
Such property will have to have name.

Reply:
Good, then we'll stick with pointer equality, see comment above.